config/config.exs

```diff
@@ -29,7 +29,7 @@ config :plug, :statuses, %{
   460 => "Invalid device token"
 }
 
-lager_formater_config = [:date, 'T', :time, :color, ' [', :severity, '] ', :pid, ' ', :message, '\e[0m\r\n']
+lager_formater_config = [:date, 'T', :time, ' [', :severity, '] ', :pid, ' ', :message, '\e[0m\r\n']
 config :lager,
   colored: true,
```

Comment:
```suggestion
  colored: false,
```

Reply:
Doesn't the change above fix the issue instead? If this doesn't remove the escape sequence, I still think we should do both.

Reply:
@rslota done :)